content/03.use-cases-a.md

```diff
@@ -13,34 +13,34 @@ Version control is particularly useful if a mistake has been made where a user h
+GitHub provides a large variety of uses for hosting teaching/educational materials.
+In fact, through taking advantage of the suite of GitHub features, the entire process of running a course, workshop, or even just a lecture, can all be done openly on GitHub.
+As a matter of gross simplification, organizing a course (for example) could be broken down into: 1) developing the material (i.e., slides, examples, relevant readings, labs, etc.), 2) hosting the course on some online platform for students to access, 3) delivering the content, and 4) accepting student work submissions and then returning graded material.
 While of course there are other purpose-built platforms for this type of activity, few of them provide the usability at the price point GitHub does. 
 
-First, developing course material, from slides to labs and everything in between, can be done on GitHub, out in the open, where others can see, review and offer feedback on your process!  
-Making presentations can be done through most major high-level programming languages such as [R, with `RMarkdown`](https://rmarkdown.rstudio.com/lesson-11.html), [Python, with `python-ppt`](https://python-pptx.readthedocs.io/en/latest/index.html), and [Julia, with `Remark.jl`](https://juliapackages.com/p/remark).  
-Since all these programs work via code bases, they can be version-controlled through git and GitHub.  
-Once you've made all the content for your course, hosting a course website can be done through GitHub pages, and there are [lots of templates available](https://github.com/topics/course-website).  
-This way, not only can the course content be available to your enrolled students, but also to a global pool of learners and teachers interested in the course material.  
-Since the course material can be easily housed on a GitHub pages website, it is then simple enough to deliver the content via that website, and/or a GitHub organization with template repositories for assignments etc.  
-Student submissions are perhaps the least seamless component, but for assignments submitted as code files (i.e., `.R` & `.Rmd` as two of the most common) and/or `.pdf` files, GitHub has a new and far-from-perfect but still useful tool [GitHub classroom](https://classroom.github.com/) where instructors can host private assignments, and even build custom autograding tests, that will autograde assignments!
+First, developing course material, from slides to labs and everything in between, can be done on GitHub, out in the open, where others can see, review and offer feedback on your process.
+Making presentations can be done through most major high-level programming languages such as R, with `RMarkdown` [@https://rmarkdown.rstudio.com/lesson-11.html], Python, with `python-ppt` [@https://python-pptx.readthedocs.io/en/latest/index.html]), and Julia, with `Remark.jl` [@https://juliapackages.com/p/remark].
```

Comment:
Official citation for RMarkdown from `citation("rmarkdown")`
>Yihui Xie and J.J. Allaire and Garrett Grolemund (2018). R Markdown: The
  Definitive Guide. Chapman and Hall/CRC. ISBN 9781138359338. URL
  https://bookdown.org/yihui/rmarkdown.